scripts/dockerfile_checks.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+fromGolangLines="$(grep 'FROM golang' Dockerfile | wc -l)"
```

Comment:
Instead of verifying the exact textual contents of the dockerfile, what do you think about just running dockerised scip-go on its own sources? 

Because really that's what we care about - whether toolchain is present and the binary is workable

Reply:
I think let's leave these checks in for now, we can easily remove them if they're annoying/are hitting false positives. I've added a check for running `scip-go` on itself as you suggested, as I agree that is a good idea.